Mu2eUtilities/inc/simParticleList.hh

```diff
@@ -33,6 +33,13 @@ namespace mu2e {
     return simParticleList(simh, PDGCode::mu_minus, ProcessCode::muMinusCaptureAtRest);
   }
 
+
+inline std::vector<art::Ptr<SimParticle> > stoppedMuPlusList(art::ValidHandle<SimParticleCollection> simh) {
+    // G4 sets this end code 
+  return simParticleList(simh, PDGCode::mu_plus, ProcessCode::Decay);
```

Comment:
How do we avoid resampling muons that decay in flight? Presumably they have the same ProcessCode, or am I missing something?

Reply:
I agree with Andy.  We follow the convention that if a process  is something that Mu2e controls, then the name of the ProcessCode must contain the string "mu2e"; all other process codes are the name of a Geant4 G4VProcess.    So you should add a new process code, maybe, mu2eMuPlusDecayAtRest ?

Reply:
I am not sure if this is the right approach. I think in this case Geant4 controls the decay of the muplus. I am not quite sure how to avoid muons stopped else where.

Reply:
This is a question to Andy and Rob due to me lack the knowledge the answer; 
I wonder if in the source file, when resampling from the stopped particles, something like this filter being applied;
"mu2e::SimParticlemv_TargetStopFilter"
Or if it is a cut implemented on the art file of the stopped particles, so that only those stopped in the stopping target were saved?

Reply:
I read deeper into the good and I realize now that I misunderstood how this works.  I had thought that you were setting a creation ProcessCode but I see now that it is testing the already existing end ProcessCode.  So I take back my comment.

However Andy's comment is still important.  Do you want only only muons that decay at rest in the stopping target?  Do you want only muons that decay at rest in any material?  Do you want muons that decay in flight anywhere in the apparatus?

I don't know what cuts went into making the input file.  Andy do you?

Reply:
I think this module would only need muons decay at rest in the stopping target.